apps/ui-storybook-e2e/src/integration/popover/popover.cy.ts

```
@@ -95,3 +95,63 @@ describe('popover--default', () => {
+			});
+		});
+
+	/**
+	 * Cypress scrolls by assigning `scrollTop`, which never reaches the listener
+	 * `ScrollDispatcher` installs, so the reposition strategy would not run. Dispatching the
+	 * event explicitly is what makes this a scroll as far as the overlay is concerned.
+	 */
+	const scrollTo = (y: number) => {
+		cy.scrollTo(0, y);
+		cy.document().then((doc) => doc.dispatchEvent(new Event('scroll')));
+		cy.wait(100);
```

ESLint

[error] Do not wait for arbitrary time periods

(cypress/no-unnecessary-waiting)